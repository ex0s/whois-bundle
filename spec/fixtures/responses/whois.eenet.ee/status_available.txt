
The registry database contains only .EE, .COM.EE, .PRI.EE, .FIE.EE,
.ORG.EE and .MED.EE domains.

NOT FOUND: u34jedzcq.ee

