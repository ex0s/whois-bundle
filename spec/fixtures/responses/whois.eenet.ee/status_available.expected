#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s == []
